magellan/tests/test_sampler_down_sample.py

```diff
@@ -0,0 +1,120 @@
+# coding=utf-8
+import sys
+import magellan
+import os
+from nose.tools import *
+import unittest
+import pandas as pd
+import six
+
+from magellan.utils.generic_helper import get_install_path
+from magellan.sampler.down_sample import _inv_index, _probe_index, down_sample, _get_str_cols_list
+import magellan.catalog.catalog_manager as cm
+from magellan.io.parsers import read_csv_metadata
+
+datasets_path = os.sep.join([get_install_path(), 'datasets', 'test_datasets'])
+path_a = os.sep.join([datasets_path, 'restA.csv'])
+path_b = os.sep.join([datasets_path, 'restB.csv'])
+
+class DownSampleTestCases(unittest.TestCase):
+    def test_down_sample_table_valid_1(self):
+        A = read_csv_metadata(path_a)
+        B = read_csv_metadata(path_b, key='ID')
```

Comment:
Why not add a setUp() function for the class and then read the two tables there? See test_attr_equiv_blocker.py for reference.